### PR TITLE
FEAT: Support gptq quantization

### PR DESCRIPTION
**Pull request by codingl2k1**
- Add https://huggingface.co/Qwen/Qwen-7B-Chat-Int4
- Add https://huggingface.co/Qwen/Qwen-7B-Chat-Int8
- Add https://huggingface.co/Qwen/Qwen-14B-Chat-Int4
- Add https://huggingface.co/Qwen/Qwen-14B-Chat-Int8
- Add https://modelscope.cn/models/qwen/Qwen-7B-Chat-Int4/summary
- Add https://modelscope.cn/models/qwen/Qwen-7B-Chat-Int8/summary
- Add https://modelscope.cn/models/qwen/Qwen-14B-Chat-Int4/summary
- Add https://modelscope.cn/models/qwen/Qwen-14B-Chat-Int8/summary

cmdline example: `xinference launch -n qwen-chat -f gptq -s 7 -q Int4`

### CODE REVIEW

**Comment by aresnow1** [comment]
These models are also supported by modelscope.

**Comment by codingl2k1** [comment]
> These models are also supported by modelscope.

OK. Modelscope support is included in this PR.

**Comment by aresnow1** [comment]
Have you checked if it works on UI?

**Comment by codingl2k1** [comment]
> Have you checked if it works on UI?

I have checked gptq from both sources, the huggingface and modelscope.

File: xinference/model/llm/llm_family.py [hunk 1]
```diff
@@ -650,6 +650,11 @@ def _match_quantization(q: Union[str, None], quantizations: List[str]):
             if q.lower() == quant.lower():
                 return quant
 
+    def _apply_format_to_model_id(spec: LLMSpecV1, q: str) -> LLMSpecV1:
+        if "{" in spec.model_id:
```
Comment (ChengjieLi28) [inline xinference/model/llm/llm_family.py:654]: Add a comment here.

Comment (codingl2k1) [reply xinference/model/llm/llm_family.py:656]: Comments added. Thanks.

**Review by ChengjieLi28 — APPROVED** [review]

**Pull request merged**